### PR TITLE
refactor: break up the Travel page into several subcomponents, and move const strings into CONSTANTS file

### PR DESCRIPTION
**Pull request by pierreTklein**
### List of changes:

- break up the Travel page into several subcomponents
- move const strings into CONSTANTS file

### Type of change:

Please delete options that aren't relevant.

- [X] Bug fix (non-breaking change which fixes an issue)

### PR Checklist:

- [X] Merged `develop` branch (before testing)
- [X] Linted my code locally
- [ ] Listed change(s) in the Changelog
- [X] Tested all links in project relevant browsers
- [X] Tested all links on different screen sizes
- [X] Referenced all useful info (issues, tasks, etc)

### CODE REVIEW

File: src/pages/Travel/TravelStatusBus.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+interface ITravelStatusProps {
+  travel: ITravel;
+}
+
+const TravelStatusBus: React.FunctionComponent<ITravelStatusProps> = ({
+  travel,
+}) => {
+  return (
+    <div>
+      Congratulations, you've secured a seat on our Toronto bus to/from McHacks!
```
Comment (logan-r) [inline src/pages/Travel/TravelStatusBus.tsx:20]: nit: "Congratulations, you've secured a seat on our {BUS_DEPARTURE_CITY} bus to/from McHacks!"

File: src/config/constants.ts [hunk 2]
```diff
@@ -20,6 +17,15 @@ export const HACKATHON_DATE = 'February 1-2, 2020';
 export const HACKATHON_LOCATION = 'McGill University';
 export const STATIC_SITE = 'https://mchacks.ca';
 
+// Travel information
+export const BUS_DEPARTURE_CITY = 'Toronto';
```
Comment (logan-r) [inline src/config/constants.ts:21]: Outside the scope of this PR, but we should definitely restructure this eventually to support multiple buses.

**Review by logan-r — CHANGES_REQUESTED** [review]

File: src/config/constants.ts (hunk 2 above)
Comment (pierreTklein) [reply src/config/constants.ts:21]: This would be within the Travel information schema.

File: src/pages/Travel/TravelStatusBus.tsx (hunk 1 above)
Comment (pierreTklein) [reply src/pages/Travel/TravelStatusBus.tsx]: Done

**Pull request merged**